digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/RoadLinkService.scala

```diff
@@ -227,6 +227,9 @@ class RoadLinkService(val vvhClient: VVHClient, val eventbus: DigiroadEventBus,
   def getRoadLinksFromVVH(bounds: BoundingRectangle, bounds2: BoundingRectangle) : Seq[RoadLink] =
     getRoadLinksAndChangesFromVVH(bounds, bounds2)._1
 
+  def getRoadLinksFromVVHByBounds(bounds: BoundingRectangle, bounds2: BoundingRectangle, newTransaction: Boolean = true) : Seq[RoadLink] =
+    getRoadLinksAndChangesByBoundsFromVVH(bounds, bounds2, newTransaction)._1
```

Comment:
probably this method "getRoadLinksFromVVHByBounds" could be merged with "getRoadLinksFromVVH"

Reply:
It's not possible to do the overload of this method, since it has an default parameter and there are already an overload of it with a default.